### PR TITLE
new order created once cart is purchased

### PR DESCRIPTION
**Pull request by pal-16**

### CODE REVIEW

File: client/src/components/Cart/Total.js [hunk 1]
```diff
@@ -51,9 +51,8 @@ const Total = (props) => {
   const items = props.items;
   const title = `(Rs.${props.price})`;
   const classes = CartStyles();
-
+  const state = useAuthState();
```
Comment (ShubhankarKG) [inline client/src/components/Cart/Total.js:54]: We are not using this state anywhere. If it's not needed, we should remove this.

**Review by ShubhankarKG — CHANGES_REQUESTED** [review]

File: client/src/components/Cart/Total.js (hunk 1 above)
Comment (pal-16) [reply client/src/components/Cart/Total.js:54]: We are using it

Comment (pal-16) [reply client/src/components/Cart/Total.js:54]: https://github.com/DSC-VJTI/CA-Website/blob/main/client/src/components/Cart/Total.js
Line 70

I got an error for the same that state is not defined, when I was trying to run the client side, using state solved it as well

Comment (ShubhankarKG) [reply client/src/components/Cart/Total.js:54]: Okay, never noticed that. Works

**Review by ShubhankarKG — APPROVED** [review]

**Pull request merged**